shards/modules/gfx/steps.cpp

```diff
@@ -264,17 +264,26 @@ struct DrawablePassShard {
   static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
   static SHTypesInfo outputTypes() { return ShardsTypes::PipelineStep; }
 
+  static SHOptionalString help() {
+    return SHCCSTR("This shard creates a render pass object, meant for rendering drawable objects as is, using the drawables "
+                   "queue object (specified in the Queue parameter) "
+                   "and the sequence of features objects (specified in the Features parameter).");
+  }
+
+  static SHOptionalString inputHelp() { return DefaultHelpText::InputHelpIgnored; }
+  static SHOptionalString outputHelp() { return SHCCSTR("The render pass object for use in a render pipeline."); }
+
   PARAM_EXT(ParamVar, _name, ShardsTypes::NameParameterInfo);
-  PARAM_PARAMVAR(_queue, "Queue", "The queue that this pass should render", {DrawQueueVarType});
+  PARAM_PARAMVAR(_queue, "Queue", "The drawables queue to use to create the render pass.", {DrawQueueVarType});
```

Comment:
Maybe better to say: "The drawable queue to get drawables from" or "The drawable queue to draw objects from when rendering this pass"